src/openforms/prefill/validators.py

```diff
@@ -0,0 +1,34 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+from openforms.authentication.registry import register as auth_register
+from openforms.forms.validation.base import BaseValidator
+from openforms.forms.validation.registry import register
+from openforms.typing import JSONObject
+
+from .co_sign import get_default_plugin_for_auth_attribute
+
+
+@register("coSign")
+class CoSignValidator(BaseValidator):
+    def __call__(self, component: JSONObject) -> None:
+        if not (auth_plugin_id := component.get("authPlugin")):
+            raise ValidationError(
+                _(
+                    "The co-sign component is missing the authPlugin configuration option"
+                ),
+                code="invalid",
+            )
+
+        auth_plugin = auth_register[auth_plugin_id]
+        default_plugin = get_default_plugin_for_auth_attribute(
+            auth_plugin.provides_auth
+        )
+        if not default_plugin:
+            raise ValidationError(
+                _(
+                    "The co-sign component would look up the name by '{attr}', but the "
+                    "global prefill default is not (yet) configured for this."
```

Comment:
When I was clicking through the interface I found the error a bit confusing. Maybe it could be changed to something along the lines of:
`The co-sign component requires the Default person plugin (Global Prefill configuration) to be configured.`

Reply:
yeah it didn't sit right and I hoped it would have been caught with the translations :D but I can modify it to that, good suggestion!